### ISSUE TITLE
We cannot properly reproduce the results in the paper

### ISSUE THREAD

**Issue by zhujiem** [DeepGraphLearning/RecommenderSystems#3]
Could you please share us all the parameters used? Thanks!

**Comment by zhujiem** [comment]
For avazu and criteo datasets.

**Comment by shichence** [comment]
For which model? Autoint or Autoint+? What does your AUC and logloss look like?

**Comment by shichence** [comment]
Autoint for avazu: {'blocks': 3, 'block_shape': [64, 64, 64], 'heads': 2, 'embedding_size': 16, 'dropout_keep_prob': [1, 1, 0.5], 'epoch': 1, 'batch_size': 1024, 'learning_rate': 0.001, 'learning_rate_wide': 0.01, 'optimizer_type': 'adam', 'l2_reg': 0.0, 'random_seed': 2018, 'save_path': './model/Avazu/b3h2_64x64x64/', 'field_size': 23, 'loss_type': 'logloss', 'verbose': 1, 'run_times': 10, 'is_save': True, 'greater_is_better': False, 'has_residual': True, 'has_wide': False, 'deep_layers': None, 'batch_norm': 0, 'batch_norm_decay': 0.995, 'data': 'Avazu'}

**Comment by shichence** [comment]
AutoInt for Criteo: {'blocks': 3, 'block_shape': [64, 64, 64], 'heads': 2, 'embedding_size': 16, 'dropout_keep_prob': [1, 1, 0.5], 'epoch': 2, 'batch_size': 1024, 'learning_rate': 0.001, 'learning_rate_wide': 0.01, 'optimizer_type': 'adam', 'l2_reg': 0.0, 'random_seed': 2018, 'save_path': './model/Criteo/b3h2_64x64x64/', 'field_size': 39, 'loss_type': 'logloss', 'verbose': 1, 'run_times': 5, 'is_save': True, 'greater_is_better': False, 'has_residual': True, 'has_wide': False, 'deep_layers': None, 'batch_norm': 0, 'batch_norm_decay': 0.995, 'data': 'Criteo'}

**Comment by zhujiem** [comment]
Thank you very much for you quick reply. I try AutoInt+, but I use my own split according to 8:1:1. I will try it again with your parameters. Would you also share your code for data splitting?  I found that random splits with different seeds can result in quite different results.

**Comment by shichence** [comment]
For data splitting, please refer to the README and we provide script in `data`. Do not forget to rescale the numerical data.

**Comment by JinYang88** [comment]
Could you please also share the parameters for autoint+ on Avazu and Criteo? 

BTW, did you try different configures for embedding_size and batch_size?

**Comment by zhujiem** [comment]
> For data splitting, please refer to the README and we provide script in `data`. Do not forget to rescale the numerical data.

Oh, I found it. Thanks! Do you also use random_seed=2018 for avazu splitting?

**Comment by zhujiem** [comment]
> For which model? Autoint or Autoint+? What does your AUC and logloss look like?

I just found the paras are for AutoInt. Could you please share the paras for AutoInt+?

**Comment by shichence** [comment]
> > For data splitting, please refer to the README and we provide script in `data`. Do not forget to rescale the numerical data.
> 
> Oh, I found it. Thanks! Do you also use random_seed=2018 for avazu splitting?

To be honest,  we didn't set random seed for data splitting

**Comment by shichence** [comment]
> Could you please also share the parameters for autoint+ on Avazu and Criteo?
> 
> BTW, did you try different configures for embedding_size and batch_size?

We have only tried embeddings of 16 due to the limited cuda memory. For `batch_size`, I recommend `1024`, which will give the best results. Note that all the baselines in our paper were run with fixed batch size 1024.

**Comment by shichence** [comment]
Autoint+ for Criteo (Please ignore the args that are not in the released code)

{'blocks': 3, 'block_shape': [64, 64, 64], 'heads': 2, 'embedding_size': 16, 'dropout_keep_prob': [1.0, 1.0, 1.0], 'epoch': 2, 'batch_size': 1024, 'learning_rate': 0.001, 'learning_rate_wide': 0.01, 'optimizer_type': 'adam', 'l2_reg': 0.0, 'random_seed': 2018, 'save_path': './model/Criteo/b3h2_64x64x64_dnn400x2/', 'field_size': 39, 'loss_type': 'logloss', 'verbose': 1, 'run_times': 10, 'is_save': True, 'greater_is_better': False, 'has_residual': True, 'has_wide': False, 'deep_layers': [400, 400], 'batch_norm': 0, 'batch_norm_decay': 0.995, 'data': 'Criteo'}

**Comment by shichence** [comment]
Autoint+ for Avazu (Please ignore the args that are not in the released code)

{'blocks': 3, 'block_shape': [64, 64, 64], 'heads': 2, 'embedding_size': 16, 'dropout_keep_prob': [1.0, 1.0, 1.0], 'epoch': 1, 'batch_size': 1024, 'learning_rate': 0.001, 'learning_rate_wide': 0.01, 'optimizer_type': 'adam', 'l2_reg': 0.0, 'random_seed': 2018, 'save_path': './model/Avazu/b3h2_64x64x64_dnn400x2/', 'field_size': 23, 'loss_type': 'logloss', 'verbose': 1, 'run_times': 10, 'is_save': True, 'greater_is_better': False, 'has_residual': True, 'has_wide': False, 'deep_layers': [400, 400], 'batch_norm': 0, 'batch_norm_decay': 0.995, 'data': 'Avazu'}

**Comment by PhenixY** [comment]
> For which model? Autoint or Autoint+? What does your AUC and logloss look like?

Where is the code of Autoint+ ?  What's the difference between Autoint and Autoint+? Sorry I only find the code for autoint

**Comment by PhenixY** [comment]
> > For which model? Autoint or Autoint+? What does your AUC and logloss look like?
> 
> I just found the paras are for AutoInt. Could you please share the paras for AutoInt+?

Hi, where is AutoInt+ ? the github code is only for autoint, right?

**Comment by PhenixY** [comment]
I  still can not reproduce the result in paper as the above parameters, anyone can help me?
